test/geometry_test_common.hpp

```diff
@@ -212,19 +215,34 @@ inline void BoostGeometryWriteTestConfiguration()
 #ifdef BOOST_GEOMETRY_TEST_FAILURES
 #define BG_NO_FAILURES 0
 inline void BoostGeometryWriteExpectedFailures(std::size_t for_rescaling,
-                std::size_t for_no_rescaling = BG_NO_FAILURES)
+                std::size_t for_no_rescaling_double,
+                std::size_t for_no_rescaling_float,
+                std::size_t for_no_rescaling_extended)
 {
-    boost::ignore_unused(for_rescaling, for_no_rescaling);
+    boost::ignore_unused(for_rescaling, for_no_rescaling_double,
+                         for_no_rescaling_float, for_no_rescaling_extended);
+
+    std::size_t const for_no_rescaling
+        = if_typed<default_test_type, double>(for_no_rescaling_double,
+              if_typed<default_test_type, float>(for_no_rescaling_float,
+                  for_no_rescaling_extended));
 
-#if defined(BOOST_GEOMETRY_TEST_ONLY_ONE_TYPE) && defined(BOOST_GEOMETRY_TEST_ONLY_ONE_ORDER)
-    std::cout << std::endl;
 #if defined(BOOST_GEOMETRY_USE_RESCALING)
     std::cout << "RESCALED - Expected: " << for_rescaling << " error(s)" << std::endl;
-#else
+#elif defined(BOOST_GEOMETRY_TEST_ONLY_ONE_TYPE) && defined(BOOST_GEOMETRY_TEST_ONLY_ONE_ORDER)
     std::cout << "NOT RESCALED - Expected: " << for_no_rescaling << " error(s)" << std::endl;
-#endif
+#else
+    std::cout << std::endl;
 #endif
 }
+
+inline void BoostGeometryWriteExpectedFailures(std::size_t for_rescaling,
+                std::size_t for_no_rescaling_double = BG_NO_FAILURES)
+{
+  BoostGeometryWriteExpectedFailures(for_rescaling, for_no_rescaling_double,
```

Comment:
also here I see an indentation issue

Reply:
oh yes, sorry about this.
I'm working on another machine now with the default 2 (which is actually **much** better but not the boost convention).
I'll fix it later. Thanks.